src/ContentValidationListener.php

```diff
@@ -82,7 +88,7 @@ public function __construct(
      * @param EventManagerInterface $events
      * @param int $priority
      */
-    public function attach(EventManagerInterface $events, $priority = 1)
+    public function attach(EventManagerInterface $events)
```

Comment:
The above change should be reverted; it's present to keep signature compatibility with the `ListenerAggregateInterface`.